amundsen_application/static/js/ducks/lastIndexed/reducer.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  GetLastIndexed,
+  GetLastIndexedRequest,
+  GetLastIndexedResponse,
+} from './types';
+
+export interface LastIndexedReducerState {
+  lastIndexed: number;
+}
+export const initialState: LastIndexedReducerState = {
+  lastIndexed: null,
+};
+
+/* ACTIONS */
+export function getLastIndexed(): GetLastIndexedRequest {
+  return { type: GetLastIndexed.REQUEST };
+}
+
+export function getLastIndexedFailure(): GetLastIndexedResponse {
+  return { type: GetLastIndexed.FAILURE };
+}
+
+export function getLastIndexedSuccess(
```

Comment:
I would create an export an interface for the payload.
Check here an example: https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/dashboard/types.ts#L23